### PR TITLE
Feature toggle sidebar visibility

### PR DESCRIPTION
**Pull request by alexweber**
@Thomas101 Would you mind reviewing this work-in-progress effort to enable toggling the sidebar? Thanks! :)

Note: I have no clue how to actually show/hide the sidebar, settings implementation & menu link _should_ be fine though

Note2: "Settings" and "Add Mailbox" would need to be added to the main app menu as well, however I couldn't figure out how to do that since they're deeply tied to the Sidelist display; perhaps a refactor is in order?

Refs #58

### CODE REVIEW

File: src/main/WMailWindow.js [hunk 1]
```diff
@@ -157,6 +160,14 @@ class WMailWindow extends EventEmitter {
   }
 
   /**
+  * Toggles the sidebar
+  */
+  toggleSidebar () {
```
Comment (alexweber) [inline src/main/WMailWindow.js]: @Thomas101 Went out on a limb with this, not sure if it's the correct approach...

**Comment by Thomas101** [comment]
Hi Alex, thanks for this. That's brilliant! I can add a bit of style jiggling to get the sidebar to show. I'll need to move the settings into the app view and then allow that to be triggered from the main process.

As regards the settings, the localstorage instance is different between the main process and renderer process, but the renderer process does copy the settings across on change. I'll update fa042a43f2a3a5575dc9086443be8970d2904659 to use this.

I've got a lot on this week but will merge it in when I get some time to do some more development! Thanks!!

**Comment by alexweber** [comment]
@Thomas101 Awesome, made my day! :)  Glad to help, will try to tackle a few others as time permits...

**Comment by Thomas101** [comment]
I've merged this in with the latest commit. Linked all the loose ends together and it should be working :+1:  Thanks!

**Comment by alexweber** [comment]
Awesome! :+1:

**Pull request merged**